integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonTableCompactor.scala

```diff
@@ -272,6 +284,13 @@ class CarbonTableCompactor(carbonLoadModel: CarbonLoadModel,
         mergedLoadName)
       OperationListenerBus.getInstance()
         .fireEvent(compactionLoadStatusPostEvent, operationContext)
+      if (null != tableDataMaps) {
+        val buildDataMapPostExecutionEvent: BuildDataMapPostExecutionEvent =
+          new BuildDataMapPostExecutionEvent(sqlContext.sparkSession,
```

Comment:
Above LoadDataMapPreExecutionEvent is used and here BuildDataMapPostExecutionEvent is used. Check if this is correct. I think above it should be BuildDataMapPreExecutionEvent OR here it should be LoadDataMapPostExecutionEvent

Reply:
Changed to BuildDataMapPreExecutionEvent

Reply:
Fixed changed to BuildDataMapPreExecutionEvent .